src/enums/taskStatus.ts

```diff
@@ -0,0 +1,10 @@
+export enum TaskStatus {
+  IN_PROGRESS = 'IN_PROGRESS',
+  COMPLETED = 'COMPLETED',
+  ABORTED = 'ABORTED',
+  PAUSED = 'PAUSED',
+  FAILED = 'FAILED',
+  PENDING = 'PENDING',
+  EXPIRED = 'EXPIRED',
+  ARCHIVED = 'ARCHIVED',
```

Comment:
please explain each status for example what does "archived" mean? "expired" mean?

Reply:
ARCHIVED has been removed , duplicate meaning.